### PR TITLE
Adds boilerplate code for new current location samples.

### PR DESCRIPTION
**Pull request by codingjeremy**
Just boilerplate top-level code for new samples. Will add actual code for the APIs in a followup PR.

### CODE REVIEW

**Review by ggfan — APPROVED** [review]
LGTM

File: CurrentLocationKotlin/README.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+To run this sample, **location must be enabled**.
+
+**IMPORTANT NOTE**: You should generally prefer 'while-in-use' for location requests.
+
+Prerequisites
+--------------
+
+- Android API Level >v27
+- Google Support Repository
```
Comment (tiembo) [inline CurrentLocationKotlin/README.md:31]: is this still needed?

File: CurrentLocationKotlin/README.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+Getting Started
+---------------
+This sample uses the Gradle build system. To build this project, use the "gradlew build" command
+or use "Import Project" in Android Studio.
+
+The sample also work on Android 11, but you will need to change the build.gradle to target 30.
```
Comment (tiembo) [inline CurrentLocationKotlin/README.md:38]: nit: "the build.gradle" to `build.gradle`

File: CurrentLocationKotlin/README.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+then unsubscribe once you retrieve the location).
+
+Users have four options in Android 11+ regarding location:
+* One time only
+* Allow while app is in use, i.e., while app is in foreground
+* Allow all the time
+* Not allow location at all
+
+In this samples case, we only ask for 'while-in-use' location access via the FINE location permission
+(`android.permission.ACCESS_FINE_LOCATION`), so the **All the time** access will not show up as an
```
Comment (tiembo) [inline CurrentLocationKotlin/README.md:17]: should we add links to DAC for `ACCESS_FINE_LOCATION` and other places?

File: CurrentLocationKotlin/app/build.gradle [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+ * limitations under the License.
+ */
+plugins {
+    id 'com.android.application'
+    id 'kotlin-android'
+}
+
+android {
+    compileSdkVersion 30
+    buildToolsVersion "30.0.2"
```
Comment (tiembo) [inline CurrentLocationKotlin/app/build.gradle:23]: i don't think this is needed anymore

**Review by tiembo — CHANGES_REQUESTED** [review]

File: CurrentLocationKotlin/README.md (hunk 1 above)
Comment (codingjeremy) [reply CurrentLocationKotlin/README.md:31]: Probably not. :D Removed.

File: CurrentLocationKotlin/app/build.gradle (hunk 4 above)
Comment (codingjeremy) [reply CurrentLocationKotlin/app/build.gradle:23]: You are right. Studio still adds it. :/

File: CurrentLocationKotlin/README.md (hunk 3 above)
Comment (codingjeremy) [reply CurrentLocationKotlin/README.md:17]: Added link and another link to a guide.

**Review by codingjeremy — COMMENTED** [review]
Addressed comments.

**Review by tiembo — APPROVED** [review]

**Pull request merged**